clients/cobol-lsp-vscode-extension/src/services/copybook/CopybookDownloadService.ts

```diff
@@ -53,10 +55,10 @@ export class CopybookDownloadService implements vscode.Disposable {
     return true;
   }
 
-  private static completedDownload: number = 0;
-  private static totalDownload: number = 0;
+  public completedDownload: number = 0;
+  public totalDownload: number = 0;
 
-  private static processDownloadError(title: string) {
+  public processDownloadError(title: string) {
```

Comment:
Looks like bunch of these `private static` to `public` changes are not needed, please clean it up, so that only relevant changes remain.

Reply:
should be resolved : 9be6c0a07652acd9800635c19569de87906bf6ce